pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,105 @@
+<Remember>
+
+If you did not follow the previous tutorial and do not have an `asyncapi.yaml` file ready, then generate one by running: 
+`asyncapi new --example=tutorial.yml --no-tty`.
+
+Next, generate a server by running:
+
+    asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto
+    cd output && npm install
+
+</Remember>
+
+Now you will be validating the messages which you will be sending to your application using a Mosquitto broker and an MQTT client.
+
+## Validate messages
+In this step, you will send a message to your application using an MQTT broker and check the errors logged when you accidentally send an invalid message.
+
+1. Start your generated application.
+
+<CodeBlock language="bash">
+{`npm start`}
+</CodeBlock>
+
+2. Let's send a message:
+
+<CodeBlock language="bash">
+  {`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+</CodeBlock>
+
+Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message. You should see something displayed in the terminal similar to the following:
+
+<CodeBlock language="bash">
+  {`light/measured was received:
+{ id: 1, lumens: '3', sentAt: '2017-06-07T12:34:32.000Z' }
+❗  Message Rejected. data.lumens should be integer`}
+</CodeBlock>
+
+Here, you can see that the property `lumens` has type `integer`, but you are sending a message with type `string`.
+
+<CodeBlock language="yaml" highlightedLines={[10,11]}>
+  {`  message:
+        name: lumensInfo
+        payload:
+          type: object
+          properties:
+            id:
+              type: integer
+              minimum: 0
+              description: Id of the streetlight.
+            lumens:
+              type: integer
+              minimum: 0
+              description: Light intensity measured in lumens.`}
+</CodeBlock>
+
+3. Send a correct message to your application:
+
+<CodeBlock language="bash">
+  {`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+</CodeBlock>
+
+You can see that your generated application received a message in the terminal:
+
+<CodeBlock language="bash">
+  {`light/measured was received:
+{ id: 1, lumens: 3, sentAt: '2017-06-07T12:34:32.000Z' }`}
+</CodeBlock>
+
+This indicates that your message is valid and the application recieved it correctly.
+
+## Summary
+In this tutorial, you learned how to connect your generated application to an MQTT broker, send messages through it, identify when an invalid message is sent to your application, and how to correct an invalid message. 
+
+---
+
+<DocsButton
+  suggestions={[
+    {
+      href: '/docs/tutorials/generate-code',
+      title: 'Generate code',
+      type:'back',
+    },
+    {
+      href: '/docs/tools',

```

Comment:
As far as I can see in the Docs tree presented in the sidebar, it is not aligned properly. If you want to make the next button point towards the `Tools - Overview`, kindly change the `weight` parameter accordingly.